lib/jake/array.ex

```diff
@@ -1,5 +1,100 @@
 defmodule Jake.Array do
-  def gen(_) do
-    StreamData.constant([])
+  @type_list [
+    %{"type" => "integer"},
+    %{"type" => "number"},
+    %{"type" => "boolean"},
+    %{"type" => "string"},
+    %{"type" => "null"},
+    nil
+  ]
+
+  @min_items 0
+
+  @max_items 1000
+
+  def gen_array(map, type), do: arraytype(map, map["enum"], map["items"])
```

Comment:
May be we can name things by what they are about. E.g. Array could be about [list or typle](https://json-schema.org/understanding-json-schema/reference/array.html#items). The code seems to handle both cases, but probably the cases could be named better?

E.g. Rename arraytype methods to either `list_gen` or `tuple_gen` based on the case handled.
```elixir
def gen_array(%{ "items": items}=map, _) do
  cond do
    is_map(items) -> list_gen(map, items) # note: not passing map["enum"], as mentioned in previous comment
    is_array(items) or items == nil -> tuple_gen(map, items)
    _ -> raise error # ?
  end
end
end
```

Reply:
I agree about naming things properly. It will avoid unnecessary ambiguity .